app/controllers/backoffice/categories_controller.rb

```diff
@@ -1,5 +1,6 @@
 module Backoffice
   class CategoriesController < ApplicationController
+    before_action authenticate_user!
```

Comment:
preciso fazer um before_action authorize_user! para conferir que o user é do tipo admin. lembra em que arquivo eu faço esse helper? eu estouro um erro e mando o user pra root? preciso de ideias @emilyfaccin

Reply:
Eu to fazendo um concern no MR #63 que vai filtrar não admins dos controllers.
Assim que o código entrar na main dá pra usar.